### ISSUE TITLE
Configure Plugin – Tool Bar Groups Lack Accessible Names

### ISSUE THREAD

**Issue by kimisgold** [omeka/Omeka#1120]
"Description: On the Configure Plugin page, it was found that the tool bar groups lack accessible names. When first navigating to the tool bar with arrow keys it is announced as ‘application clickable’, this should have a name such as ‘Formatting tool bar’. Then upon navigating over the ‘tool bar groups’ the groups are all announced as ‘tool bar’ instead of having additional context such as ‘Text formatting’ for the bold and underline button section.  

Note: While the first button within each group is announced, this does not give the user the clear expectation of more than one button within the group.

Note: This also occurs on the add items page after checking an HTML checkbox.

Expected Results: The tool bar groups have names to describe the tool buttons within the group.  

Actual Results: The tool bar groups are all announced as ‘Tool bar group’"

(1.3.1 Info and Relationships; Level A; WCAG 2.0)

**Comment by kimisgold** [comment]
I think this has been addressed since upgrading tinymce to version 5. I'm seeing labels for "formatting", "alignment", and "lists" toolbars. Can we retest this?

**Comment by aviannamiller** [comment]
Agreed, this is fixed.
